src/jquery.mockjax.js

```diff
@@ -667,6 +667,22 @@
 		origSettings.urlParams = paramValues;
 	}
 
+	/**
+	 * Clears handlers that mock given url
+	 * @param url
+	 * @returns {Array}
+	 */
+	function clearByUrl(url) {
+		var results = [];
+		for (var i=0, len=mockHandlers.length; i<len; i++) {
+			var handler = mockHandlers[i];
+			if (handler.url !== url) {
```

Comment:
If we also want to match regular expressions then this equality check will need a little work as strict equality won't match.